10up/insecure-content-warning
# Add better CSS to handle long URLs

### Description of the Change

In the warnings panel that displays in Gutenberg, if you have a really long URL, this isn't currently breaking properly and can cause horizontal scrolling. This PR fixes that by adding the `word-break` property.

### Alternate Designs

None

### Benefits

Better formatting when long URLs are used

### Possible Drawbacks

None

### Verification Process

1. Add an insecure image that has a long URL
2. Save the post and see the warnings panel show
3. Ensure the panel doesn't have horizontal scrolling

### Checklist:


### Changelog Entry

> Fix - ensure long URLs break properly
